### PR TITLE
DOC-425 part 5: GATK4 MuTect2 Tumor-Only Annotation, GATK4 MuTect2 Tumor-Only, MuSE Annotation, MuSE, MuTect2 Annotation, and MuTect2

### PR DESCRIPTION
**Pull request by ebusygina**
Drafts for GATK4 MuTect2 Tumor-Only Annotation, GATK4 MuTect2 Tumor-Only, MuSE Annotation, MuSE, MuTect2 Annotation, and MuTect2 workflows.
MuSE and MuTect2 have been written before but I changed them slightly so it makes sense to check them as well.

### CODE REVIEW

File: docs/Encyclopedia/pages/GATK4_MuTect2_Tumor-Only.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# GATK4 MuTect2 Tumor-Only
+
+## Description ##
+
+GATK4 MuTect2 Tumor-Only is a somatic variant calling pipeline utilized in the GDC for analyzing tumor-only whole exome sequencing (WXS) and targeted sequencing data.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/GATK4_MuTect2_Tumor-Only.md:5]: add "without normal samples" to the end of this sentence.

File: docs/Encyclopedia/pages/MuSE_Annotation.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# MuSE Annotation
+
+## Description ##
+
+MuSE Annotation is a somatic mutation annotation pipeline in GDC whole exome sequencing (WXS) and targeted sequencing harmonization. 
+
+## Overview ##
+
+MuSE Annotation is a workflow in the GDC that annotates somatic variants identified by the MuSE variant calling pipeline. Somatic mutation annotation is performed with MuSE Annotation using tumor and normal alignments and generates simple nucleotide variation (SNV) data.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/MuSE_Annotation.md:9]: "Somatic mutation annotation is performed with MuSE Annotation using tumor and normal alignments and generates simple nucleotide variation (SNV) data."  This sentence can be removed.

File: docs/Encyclopedia/pages/MuSE_Annotation.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+### Input
+
+* Tumor VCF - WXS or Targeted Sequencing
+* Normal VCF - WXS or Targeted Sequencing
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/MuSE_Annotation.md:14]: The tumor and normal BAM are compared to generate the VCF, which is effectively a comparison of the tumor and normal DNA.  I would recommend:

`* VCF (Data Type:  Raw Simple Somatic Mutation)`

File: docs/Encyclopedia/pages/MuTect2_Annotation.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# MuTect2 Annotation
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/MuTect2_Annotation.md:1]: Same comments as "MuSE Annotation"

**Review by wwysoc2 — CHANGES_REQUESTED** [review]

File: docs/Encyclopedia/pages/GATK4_MuTect2_Tumor-Only.md (hunk 1 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/GATK4_MuTect2_Tumor-Only.md]: added

File: docs/Encyclopedia/pages/MuSE_Annotation.md (hunk 2 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/MuSE_Annotation.md]: removed

File: docs/Encyclopedia/pages/MuSE_Annotation.md (hunk 3 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/MuSE_Annotation.md]: changed

File: docs/Encyclopedia/pages/MuTect2_Annotation.md (hunk 4 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/MuTect2_Annotation.md:1]: changed

**Review by wwysoc2 — APPROVED** [review]

**Pull request merged**